app/Database/Migrations/2024-09-13-152827_AddEvent.php

```diff
@@ -61,6 +61,10 @@ public function up()
                 'type' => 'TEXT',
                 'null' => false,
             ],
+            'schedule_visible_from' => [
+                'type' => 'DATETIME',
+                'null' => true,
```

Comment:
Why is it necessary to make this nullable?

Reply:
I thought was: It may be the case that we don't (yet) know when we want to publish the schedule. For that case, it can be nullable.

Reply:
Okay I understand.
We could also default it on the Event date and than change it later on.
What you like best.

Reply:
I guess we will always want to set this manually. So I'd just leave it as nullable if that's okay for your.